core-web/libs/ui/src/lib/components/dot-asset-search/components/dot-asset-card/dot-asset-card.component.ts

```diff
@@ -0,0 +1,18 @@
+import { ChangeDetectionStrategy, Component, Input, NO_ERRORS_SCHEMA } from '@angular/core';
+
+import { CardModule } from 'primeng/card';
+
+import { DotCMSContentlet } from '@dotcms/dotcms-models';
+
+@Component({
+    selector: 'dot-asset-card',
+    templateUrl: './dot-asset-card.component.html',
+    styleUrls: ['./dot-asset-card.component.scss'],
+    standalone: true,
+    imports: [CardModule],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    schemas: [NO_ERRORS_SCHEMA] // WebComponent
```

Comment:
if is only for web components, can use CUSTOM_ELEMENTS_SCHEMA?

Reply:
Done!